lib/app_web/templates/layout/app.html.eex

```diff
@@ -1,6 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-115618142-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
```

Comment:
@Cleop where is `arguments` defined...?

Reply:
The instructions on Google Analytics say this:
![image](https://user-images.githubusercontent.com/16775804/37398010-059a728a-2775-11e8-8043-b0951791826d.png)
or see:
https://analytics.google.com/analytics/web/#/a115618142w171813768p171426411/admin/tracking/tracking-code/

It looks like the arguments are where they're expecting tag manager data layer arguments (which we haven't implemented yet) and so in theory this is not being used. However I'm not sure if there are other parts of that script that we do need and so I'm hesitant to take it out based on the instructions' guidance.

Reply:
Yeah, that't the thing about using GTM, unless we are sending them "events" with `arguments` it will only track "generic" stuff ... perhaps it's worth opening a _separate_ ("_enhancement_") issue (_post MVP_) with a checklist of the _specific_ events we _want_ to track in order to derive our desired metrics ...?

Reply:
Sounds good 👍